lib/vsc/install/shared_setup.py

```diff
@@ -1382,7 +1382,7 @@ def parse_target(self, target, urltemplate=None):
         else:
             log.info('adding prospector to tests_require')
             tests_requires = new_target.setdefault('tests_require', [])
-            tests_requires.append('prospector >= 1.1.4')
+            tests_requires.append('prospector >= 1.1.6.2')
```

Comment:
we also need to inject the wht PR #97 inserted: actually revert the PR and inject it here.
we should also try to inject it as another verion , eg
```
deplinks = new_target.setdefault('dependency_links', [])
deplinks.append("git+https://github.com/stdweird/prospector#egg=prospector-1.1.6.2b")
```
(mark the `b` suffix to force a more recent version that latest downloadable version)